client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -203,6 +206,16 @@ const Organizations = () => {
           </LinkItem>
         ))}
       </List>
+      <DownloadUserListButton
+        icon="download"
+        href="/api/support/organizations/users"
+      >
+        Download User List
+      </DownloadUserListButton>
+      <p>
+        Export a list of Audit Admins and Jurisdiction Managers for recently
+        completed audits for all organizations.
+      </p>
```

Comment:
the `Organizations` list seemed like the best place to put this since the users do not strictly relate to active audits. If we keep adding miscellaneous support tools that operate at the scope of all audits/organizations we may want to break out a section specifically for those tools.

Reply:
Yeah I agree. It feels a bit odd having it be at the bottom of that list, but in the current product `Organizations` does seem to be the closest association. That being said, I wonder if we should just add a new section now, as it looks slightly out of place, and we run the risk of the next person adding a support tool without creating a new section in the name of following the existing pattern. But, just a gentle nudge, I don't feel strongly

Reply:
Added its own section in [0be290c](https://github.com/votingworks/arlo/pull/2077/commits/0be290c47651edf8421bdd57b53177bee6ac8d46). Given this is expected to be used on a computer screen (not tablet) and the potential for the existing columns to stretch pretty far down the page, this seemed like a better place to put it than in another row after "Active Audits" and "Organizations".

This probably deserves a redesign at some point eg. a separate tab

Reply:
Ooo nice, yeah I like that this takes advantage of the screen width!